README.md

```diff
@@ -146,7 +146,9 @@ Sending data to Datadog US (default) and configuring a few checks.
     datadog_config:
       tags: "mytag0, mytag1"
       log_level: INFO
-      apm_enabled: "true" # has to be set as a string
+      apm_config:
+        enabled: true
+        max_traces_per_second: 10
       logs_enabled: true  # log collection is available on agent 6
     datadog_config_ex:
```

Comment:
this whole `datadog_config_ex` section should be removed (if this example is meant for Agent 6)